src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -0,0 +1,271 @@
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
+                throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            this.SetHandle(IntPtr.Zero);
+            return true;
+        }
+    }
+
+    public partial class View
+    {
+        ///////////////////////////////////////////////////////////////////
+        // ****************** Accessiblity Attributes *******************//
+        ///////////////////////////////////////////////////////////////////
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AppendAccessibilityAttribute(string key, string val) {
+            Interop.ControlDevel.Dali_Toolkit_DevelControl_AppendAccessibilityAttribute(SwigCPtr, key, val);
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
+                throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void RemoveAccessibilityAttribute(string key) {
+            Interop.ControlDevel.Dali_Toolkit_DevelControl_RemoveAccessibilityAttribute(SwigCPtr, key);
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
+                throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void ClearAccessibilityAttributes() {
+            Interop.ControlDevel.Dali_Toolkit_DevelControl_ClearAccessibilityAttributes(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
+                throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        ///////////////////////////////////////////////////////////////////
+        // ************************** Highlight *************************//
+        ///////////////////////////////////////////////////////////////////
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool ClearAccessibilityHighlight() {
+            bool result = Interop.ControlDevel.Dali_Toolkit_DevelControl_ClearAccessibilityHighlight(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
+                throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return result;
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool GrabAccessibilityHighlight() {
+            bool result = Interop.ControlDevel.Dali_Toolkit_DevelControl_GrabAccessibilityHighlight(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
+                throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return result;
+        }
+
+        ///////////////////////////////////////////////////////////////////
+        // ****************** Accessiblity Relations ********************//
+        ///////////////////////////////////////////////////////////////////
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AppendAccessibilityRelation(View second, RelationType relation) {
```

Comment:
Could you check CA1062 rule?
https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1062